### PR TITLE
Minor development/contributing/ci doc links / additions

### PR DESCRIPTION
**Pull request by mitchcapper**
Add some more cross linking and the artifact url

### CODE REVIEW

File: DEVELOPMENT.adoc [hunk 1]
```diff
@@ -69,6 +69,7 @@ based on version information from https://search.maven.org, https://plugins.grad
 ** Output dir is `generated/antlr/`
 
 == Resources
+For automated build setup examples see our https://github.com/apple/pkl/blob/main/.circleci/[CircleCI] jobs like our https://github.com/apple/pkl/blob/main/.circleci/jobs/BuildNativeJob.pkl[BuildNativeJob.pkl] for examples of how we build Pkl automatically.
```
Comment (holzensp) [inline DEVELOPMENT.adoc:72]:
```suggestion
For automated build setup examples see our https://github.com/apple/pkl/blob/main/.circleci/[CircleCI] jobs like our https://github.com/apple/pkl/blob/main/.circleci/jobs/BuildNativeJob.pkl[BuildNativeJob.pkl], where we build Pkl automatically.
```
^^ Two "for example" instances in one sentence

File: README.adoc [hunk 2]
```diff
@@ -11,9 +11,13 @@
 :uri-pkl-examples: https://pkl-lang.org/main/current/examples.html
 :uri-installation: https://pkl-lang.org/main/current/pkl-cli/index.html#installation
 :uri-lang-reference: https://pkl-lang.org/main/current/language-reference/index.html
+:uri-ci-artifacts: https://s01.oss.sonatype.org/content/groups/public/org/pkl-lang/
+:uri-ci-pipeline: https://app.circleci.com/pipelines/github/apple/pkl
 
 A configuration as code language with rich validation and tooling.
 
+image:https://circleci.com/gh/apple/pkl.svg?style=svg["Apple", link="https://app.circleci.com/pipelines/github/apple/pkl"]
```
Comment (holzensp) [inline README.adoc:19]: Yes. Good call. Maybe to the far top?

File: README.adoc [hunk 3]
```diff
@@ -34,3 +38,8 @@ We'd love to hear from you!
 
 * Create an {uri-github-issue}[issue]
 * Ask a question on {uri-github-discussions}[GitHub Discussions]
+
+== Development
+* link:CONTRIBUTING.adoc[] for tips on pull requests and filing issues
+* link:DEVELOPMENT.adoc[] for build instructions
+* {uri-ci-artifacts}[Sonatype CI] for our ci/cd build {uri-ci-pipeline}[pipeline] artifacts/binaries of this repo and many related Pkl tools and packages
```
Comment (holzensp) [inline README.adoc:45]: Little awkward phrasing (subjective).
```suggestion
* {uri-ci-artifacts}[Sonatype Repository] for the artifacts/binaries built by our {uri-ci-pipeline}[CI pipelines] (and those of our other tools and packages repositories).
```

**Review by holzensp — CHANGES_REQUESTED** [review]
I like it. A few nits for your consideration (requesting changes for visibility).

**Comment by mitchcapper** [comment]
> Little awkward phrasing (subjective).
Not subjective definitely awkward:)  Changes all made.

**Review by holzensp — APPROVED** [review]

**Comment by holzensp** [comment]
Thanks very much for the contribution; merging when green

**Pull request merged**